core/src/main/scala/cromwell/core/simpleton/WdlValueSimpleton.scala

```diff
@@ -26,6 +27,7 @@ object WdlValueSimpleton {
   implicit class WdlValueSimplifier(wdlValue: WdlValue) {
     def simplify(name: String): Iterable[WdlValueSimpleton] = wdlValue match {
       case prim: WdlPrimitive => List(WdlValueSimpleton(name, prim))
+      case expr: WdlExpression => List(WdlValueSimpleton(name, WdlString(expr.valueString)))
```

Comment:
How is this currently being used?  I had thought that `WdlValueSimpleton`s were used only for expressions that had been fully evaluated to `WdlValue`s, in particular the cache and the job store.  It doesn't look like `WdlValueBuilder` could currently handle reversing this simplification.